libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -406,7 +416,7 @@ def checkout_update_deploy_script(self, node_obj: LogicalNode, git_tag):
         system_utils.execute_cmd(cmd=cmd)
         # Update deploy file
         string_to_replace = "waitForAllDeploymentsAvailable 120s"
-        new_string = "waitForAllDeploymentsAvailable 360s"
+        new_string = "waitForAllDeploymentsAvailable 500s"
```

Comment:
is this increased bcoz of any issue?

Reply:
yes for HA pod timeout issue CORTX-29667

Reply:
If that is the case, please add a comment that due to CORTX-29667 increased timeout, will be reverted back once the defect fixed